test/base.py

```diff
@@ -1,12 +1,28 @@
 import unittest
-
 import jsonschema
+import os, sys
 
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from genson import Schema
 
 class SchemaTestCase(unittest.TestCase):
+
+    def assertGenSchema(self, instance, options, expected):
```

Comment:
Likewise, I would prefer not to have this method. It means writing one or two extra lines in the test, but those lines make it very explicit what the test is doing.